snapwell/snapecl.py

```diff
@@ -105,156 +108,168 @@ def activeCellColumn(grid, owc_kw, x, y, z):
 def interpolate(grid, k_above, col, thresh):
     """Interpolates.  Takes grid, a index, the activeCellColumn and a threshold.
 
-       Returns owc given that k_above is index of first cell whose center is
-       above owc.
+    Returns owc given that k_above is index of first cell whose center is
+    above owc.
 
     """
     # we have crossed (OWC) the thresh border (from below)
     # this means inv_k is above OWC, whereas inv_k-1 is below
     k_below = k_above - 1
     if k_below < 0:
-        raise IndexError('Cannot interpolate down from bottom cell.')
+        raise IndexError("Cannot interpolate down from bottom cell.")
     a_idx_below = col[k_below][3]
-    a_idx_above = col[k_above][3] # recall col contains cell from bottom and up
+    a_idx_above = col[k_above][3]  # recall col contains cell from bottom and up
 
-    swat_below = col[k_below][4] # the SWAT/SGAS/etc in cell above and below OWC
+    swat_below = col[k_below][4]  # the SWAT/SGAS/etc in cell above and below OWC
     swat_above = col[k_above][4]
 
-    norm_swat_above  = swat_below - swat_above
+    norm_swat_above = swat_below - swat_above
     norm_swat_thresh = swat_below - thresh
-    norm_swat_ratio  = 1 - (norm_swat_thresh / norm_swat_above)
+    norm_swat_ratio = 1 - (norm_swat_thresh / norm_swat_above)
 
-    z_above = grid.get_xyz(active_index=a_idx_above)[2] # z coordinate of cell gives height
+    z_above = grid.get_xyz(active_index=a_idx_above)[
+        2
+    ]  # z coordinate of cell gives height
     z_below = grid.get_xyz(active_index=a_idx_below)[2]
-    z_diff = abs(z_above - z_below)                     # distance between cell centers
+    z_diff = abs(z_above - z_below)  # distance between cell centers
 
     owc = z_above + z_diff * norm_swat_ratio
 
     return owc
 
 
-
 def findOwc(grid, owc_kw, x, y, z, thresh=0.7, owc_offset=0.5):
     """Given a grid, owc_kw, x, y, z, find the OWC Z s.t. owc_kw(x,y,Z)=thresh.
 
-       This function returns a pair (owc, tvd) where owc is the approximate
-       (linear interpolated) OWC for (x,y) and tvd is the lowest cell center
-       above owc_exact - owc_offset.
+    This function returns a pair (owc, tvd) where owc is the approximate
+    (linear interpolated) OWC for (x,y) and tvd is the lowest cell center
+    above owc_exact - owc_offset.
 
     """
 
     col, idx = activeCellColumn(grid, owc_kw, x, y, z)
     if not col:
-        raise ValueError('Could not find cell column for (%.2f, %.2f, %.2f) in grid.' % (x,y,z) )
-
+        raise ValueError(
+            "Could not find cell column for (%.2f, %.2f, %.2f) in grid." % (x, y, z)
+        )
 
     current_swat = -1
     # Called inv_k because it is inv_k = nz - k
     for inv_k in range(0, len(col)):
-        if col[inv_k][3] >= 0:           # x,y,z,a,s
-            current_swat = col[inv_k][4] #       ^ ^
```

Comment:
This comment depends on formatting to make sense so I suggest rewriting it.

Reply:
Sorry for force pushing, but fixed.